### DISCUSSION TITLE
LT4 Vulkan drivers

### DISCUSSION THREAD

**Discussion by jwinarske** [OE4T/tegra-demo-distro#179] [Q&A] [answered]
Is it as simple as adding "vulkan" to DISTRO_FEATURES to add the Vulkan driver?

**Comment by madisongh** [comment, marked as answer]
You'd also need to include `vulkan-loader` in your build, either directly or as a dependency of some recipe that uses Vulkan.

**Reply by jwinarske** [reply]
Thanks!
